### DISCUSSION TITLE
Is it possible to disable the terminal statusline in gvim

### DISCUSSION THREAD

**Discussion by eggbean** [vim/vim#13592] [Q&A] [answered]
On terminal vim it's fine as I have more space, but I usually only have a little window for a gvim terminal. I would like to disable the statusline on gvim (Linux and Windows) if it is possible. Nothing I have tried works.

**Comment by ychin** [comment]
Shouldn't it not show up by default, unless you have a split? The default value for `laststatus` is 1, which would make the statusline not show up unless you have a split.

**Comment by Shougo** [comment]
You can disable statusline by `set laststatus=0`.

**Comment by eggbean** [comment, marked as answer]
I usually have `laststatus=2`. I came up with a solution that pretty much does what I want here. Cheers.

https://vi.stackexchange.com/q/43623/37532
